django_celery_results/migrations/0004_auto_20190506_0255.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
```

Comment:
would be better to remove pull from master and regen again

Reply:
I'll merge in origin/master and verify no issues.

Reply:
origin/master was up to date, and makemigrations shows no model changes detected.

Reply:
rather than pushing direct to celery repo fork it and use upstream and you fork as the origin from next PR. and remove the migrations file first and regen it with the latest model again.